From 47c274fa88fb6d84959049d2229d297129af50ee Mon Sep 17 00:00:00 2001
From: Tang Yuantian <Yuantian.Tang@freescale.com>
Date: Fri, 31 Oct 2014 13:23:19 +0800
Subject: [PATCH 046/128] cpufreq: enable cpu frequency switch on ls102x

Signed-off-by: Tang Yuantian <Yuantian.Tang@freescale.com>
[Xulin: Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto_RDS_20150206.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/mach-imx/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 630e987..5fcb57f 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -844,6 +844,7 @@ config SOC_LS1021A
 	select PCI_DOMAINS if PCI
 	select ZONE_DMA if ARM_LPAE
 	select FSL_SLEEP_FSM if PM
+	select ARCH_HAS_CPUFREQ
 
 	help
 	  This enable support for Freescale LS1021A  processor.
-- 
1.7.5.4

